portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +28,17 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> greetings = new ArrayList<>();
```

Comment:
Heads up: I recommend you make this arraylist a property of the class because the next step is to let users submit comments and append them to the arraylist.

Reply:
I see. Will do!